api/envoy/config/transport_socket/permissive/v2alpha/permissive.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.config.transport_socket.permissive.v2alpha;
+
+option java_outer_classname = "PermissiveProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.transport_socket.permissive.v2alpha";
+option go_package = "v2";
+
+// [#protodoc-title: Permissive]
+
+import "envoy/api/v2/core/base.proto";
+
+import "validate/validate.proto";
+
+// Configuration for permissive transport socket. This wraps transport sockets, providing the
+// ability to fall back from primary to secondary transport socket.
+message Permissive {
+  // Value indicates whether falling back from primary to secondary is allowed.
+  bool allow_fallback = 1;
```

Comment:
This is not needed, if there is no reason to configure permissive transport socket with this set to false. It should just configure with what in `primary_transport_socket` as the transport socket.